kong/db/dao/init.lua

```diff
@@ -1494,6 +1509,9 @@ function DAO:cache_key(key, arg2, arg3, arg4, arg5, ws_id)
   -- the generic code below, but building the cache key
   -- becomes a single string.format operation
   if type(key) == "string" then
+
+    key = hash_by_length(key)
```

Comment:
`hash_by_length` may not be a good name, could it be `normalize_string` or else?

Reply:
replace